File: notebooks/read_GLT.jl
using NetCDF

filename = joinpath(homedir(), "Downloads", "EMIT_L2A_RFL_001_20240521T232728_2414216_003.nc")

ds = NetCDF.open(filename)
ds

glt_x = NetCDF.getvar(ds, "glt_x")
println(glt_x)

reflectance = NetCDF.getvar(ds, "reflectance")
println(reflectance)

glt_y = NetCDF.getvar(ds, "glt_y")
println(glt_y)

File: notebooks/read_GLT.py
from os.path import join, expanduser
import numpy as np
import xarray as xr

filename = expanduser(join("~", "Downloads", "EMIT_L2A_RFL_001_20240521T232728_2414216_003.nc"))

ds = xr.open_dataset(filename)
ds

ds.dims

loc = xr.open_dataset(filename, group="location")

loc["glt_x"]

reflectance = np.array(ds["reflectance"])
print(reflectance)

glt_x = np.array(loc["glt_x"])
print(glt_x)

glt_y = np.array(loc["glt_y"])
print(glt_y) 




